server/monitor.go

```diff
@@ -78,6 +85,16 @@ func (m *Monitor) Stop() {
 	}
 }
 
+func (m *Monitor) IsEnabled() bool {
+	// The job can be disabled via an environment variable
+	val := os.Getenv("MM_TEAMS_SYNC_MONITORING_SYSTEM_DISABLED")
+	val = strings.ToLower(strings.TrimSpace(val))
+	if val == "true" || val == "1" {
+		return false
+	}
+	return true
+}
```

Comment:
Can we just add this as a proper configuration value? (This works fine for Cloud, but isn't an ideal setup if anyone else deploys.)

Reply:
I can.  This was meant as a temporary nice-to-have for developers while we move this code into a new plugin.  I can't imagine anyone else deploying this plugin.

Reply:
If we're not shipping this to production, maybe we can just live with the pain of the log spam for now? (Or are there other symptoms?)

Reply:
I have made this a plugin config setting.